Add new Tumbleweed logos

ElFrogster opened the pull request:
As a result of discussion in #10
Add all the different files for the new Tumbleweed logo

Not 100% sure about all the light-colored ones with the logo text

ElFrogster committed:
Add new Tumbleweed logos

Add all the different files for the new Tumbleweed logo

hellcp commented on Tumbleweed/square-symbolic.svg:
Would be nice to keep the symbolics 16x16

hellcp replied:
I would also make it edge to edge, to preserve the line width at 1

hellcp commented on Tumbleweed/favicon.ico:
Please make it multisize like the rest of the favicons

hellcp commented on Tumbleweed/light-inline.svg:
This is fine, though you could also make the image narrower

hellcp replied:
That's a bit overboard, let's keep padding on either side of the image

hellcp commented:
Please use `Save As -> Optimized SVG` in inkscape and optipng too

lkocman commented:
@hellcp I'm guilty of that too. Thank you for reminding us @hellcp !

ElFrogster committed:
Resize symbolic icon to 16x16

Change the size of the square-symbolic icon to 16 x 16px

ElFrogster committed:
Resize square-symbolix to 16x16

Change size of square-symbolic logo to 16 x 16px (this time for real).

ElFrogster committed:
Narrow down light-inline

Reduce width of the light-inline images so that there is only minimal space on the sides

ElFrogster committed:
Optimize SVGs

Use Inkscape's Save As..->Optimized SVG with default settings

ElFrogster commented:
The multisize favicon and optipng things I don't know how to do.

hellcp commented:
You can create multisize favicon by creating a .ico file with gimp, and creating a layer for every size. Have a look at other favicons in gimp to see how it's done there.

optipng is a command, you should be able to run it over all the pngs you are submitting

ElFrogster committed:
Make favicon multisize

Make the favicon multisize

ElFrogster committed:
Optimize PNGs

Optimize .png files with optipng command

ElFrogster commented:
Okay, I changed the favicon and pngs now. I think that should be all.

lkocman reviewed (commented):
Only one concern from my side, and this is padding. I see that you are not using any, which generally is good. But since the rest of distros are, we could have an issue if an app would be rely on that particular icon with inconsistent padding across distros.

ElFrogster commented:
Which file are you talking about specifically? I mostly tried to do things in line with what I saw elsewhere.
I removed besically all the padding on light-inline because LCP said it could be more narrow. Maybe that was overdoing it?

lkocman commented:
@ElFrogster you're correct it was the inline
![image](https://github.com/user-attachments/assets/a8dbea76-7f70-417b-9a88-e168418b9672)


If both you and @hellcp are fine with that then I won't stand in the way, but  will do the same mod for Leap instead :-)

lkocman commented:
Hello folks, any remaining blockers? I see @hellcp had some comments in the review.

lkocman commented:
@ElFrogster I'm about to submit wallpaper refresh to TW so it might be a good opportunity to finish the logo refresh too.

ElFrogster commented:
Is there anything that still needs to be done?

lkocman commented:
@ElFrogster check comments in @hellcp's review https://github.com/openSUSE/distribution-logos/pull/12/files and mark as resolved whatever is resolved. Once we're clean we're good to go I guess.

ElFrogster committed:
Make square-symbolic edge to edge

Scaled up the logo to have it fill out the entire canvas

ElFrogster committed:
Increade horizontal padding for light-inline

Add some more space on the sides for the light-inline images

ElFrogster commented:
Okay, I made the changes that were asked for.

lkocman reviewed (approved):
Fine by me. I'd prefer if @DimStar77 or @ana would merge it, since I'm not Tumbleweed Release manager. Thank you @ElFrogster !

ana commented:
Looks good to me. I don't have rights to merge this PR.

pallaswept commented:
(I realise this is already merged but #10 says to comment here, so...)

Community feedback:

https://old.reddit.com/r/openSUSE/comments/1gdixmb/ive_always_thought_the_dislike_towards_logo/

> Now it's...an infinity symbol with two arrows, but slightly more stylized.

It seems that outside of here, interpretations vary, even with the benefit of prior knowledge of the previous logo being an 'infinity symbol'. Maracas came up again. My favourite was "two bald people kissing" :rofl: 

Care factor?
